File: GAF/java/src/GramConfig.java
public class GramConfig {
    
    // Image settings
    public static final int IMAGE_SIZE = 224;
    
    // Data split options (only used if ENABLE_AUTO_SPLIT is true)
    public static final boolean ENABLE_AUTO_SPLIT = false;
    public static final float TRAIN_FRACTION = 0.60f;
    public static final float TEST_FRACTION = 0.20f;
    public static final float VALIDATE_FRACTION = 0.20f;
    
    // CSV Headers
    public static final String VOLTAGE_HEADER = "Potential (V)";
    public static final String CURRENT_HEADER = "Current (A)";
    public static final String CSV_DELIMITER = ",";
    
    // Logging
    public static final boolean VERBOSE = true;
}

File: GAF/python/gram_config.py
class GramConfig:
    # Image settings
    IMAGE_SIZE = 224
    OUTPUT_FILE_TYPE = 'png'

    # Data split options (only active when ENABLE_AUTO_SPLIT is true)
    ENABLE_AUTO_SPLIT = False
    TRAIN_FRACTION = 0.60
    TEST_FRACTION = 0.20
    VALIDATE_FRACTION = 0.20

    # CSV Headers
    VOLTAGE_HEADER = "Potential (V)"
    CURRENT_HEADER = "Current (A)"
    CSV_DELIMITER = ","

    # Logging
    VERBOSE = True
